hazelcast-sql/src/test/java/com/hazelcast/sql_slow/MapIndexScanPMigrationStressTest.java

```diff
@@ -58,15 +59,15 @@ public void before() {
             instances[i] = factory.newHazelcastInstance(smallInstanceConfig());
         }
         map = instances[0].getMap(MAP_NAME);
+        mutatorException = new AtomicReference<>(null);
     }
 
     @After
     public void after() {
-        factory.terminateAll();
+        factory.shutdownAll();
```

Comment:
Why `shutdownAll`? `terminateAll` is faster and the cluster isn't needed anymore.